src/backend/database_migrations/versions/20220803_224221_add_submitting_lab_to_groups.py

```diff
@@ -0,0 +1,32 @@
+"""add submitting lab to groups
+
+Create Date: 2022-08-03 22:42:27.655366
+
+"""
+import enumtables  # noqa: F401
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
```

Comment:
```suggestion
```

Reply:
What's the issue here? This is in all our migrations.

Reply:
i just thought it was redundant/ not useful. If you like it leave it in though!